### ISSUE TITLE
inetChecksite() is not working on Windows 

### ISSUE THREAD

**Issue by yura-chaikovsky** [sebhildebrandt/systeminformation#183]
### Issue Type

* [x] Bug
* [ ] Feature Request
* [ ] Question

### Expected behavior 

Calling `si.inetChecksite(url).then(res => {})` is expected to resolve with status and timing of request to `url`.

### Actual behavior

- `inetChecksite` resolves successfully if `url` is not available. 
- `inetChecksite` never resolves if `url` is accessible.

### Steps to reproduce the problem

```
const si = require("systeminformation");
si.inetChecksite("https://www.google.com")
    .then(response => {
        console.log(response); // <-- will never gets called
    });
```

### Specifications 

- package version: 3.51.4
- operating system: Windows 10

### Problem and possible solution

Check file `lib/internet.js` line 71:
```
69:  resolve(result);
70: } else {
71:   // res.on('data', (chunk) => {  });       
72:  res.on('end', () => {
73:    result.ms = (result.ok ? Date.now() - t : -1);
74:    if (callback) { callback(result); }
75:    resolve(result);
76:   });
```
Line 71 should be uncommented, because `end` event will not fire, if `data` is not bind. More about it here: https://stackoverflow.com/questions/23817180/node-js-response-from-http-request-not-calling-end-event-without-including-da

**Comment by sebhildebrandt** [comment]
Thank you @yura-chaikovsky ... OK, I see ...will fix it today ...
Interestingly here on my machines I never had problems so far ...

**Comment by sebhildebrandt** [comment]
Well, you where completely right! I could reproduce the problem, don't know why I did not covered this in my tests ... I now released a new version 3.52.1 with a fix for that. Thank you once again.

**Comment by yura-chaikovsky** [comment]
That was fast :) Thank *you* for quick turnaround.
